services/status.go

```diff
@@ -3,17 +3,27 @@ package services
 import (
 	"encoding/json"
 	"os"
+	"time"
 
 	"github.com/gobitfly/eth2-beaconchain-explorer/db"
 	"github.com/gobitfly/eth2-beaconchain-explorer/utils"
 	"github.com/gobitfly/eth2-beaconchain-explorer/version"
 )
 
 // Report the status of a particular service, will add current Pid and executable name
```

Comment:
this should be comment of the function, to make it easy to see in ide when using it